book/website/collaboration/academic-industry/academic-industry-community.md

```diff
@@ -0,0 +1,58 @@
+
+For the Turing-Roche Partnership this [awareness event](https://www.youtube.com/watch?v=eq7f-YnVKtg) was ran online.
+
+Later, after the first research theme was selected, the partnership ran a [series of workshops](https://www.turing.ac.uk/sites/default/files/2021-12/structured_missingness_report.pdf) which allowed an initial group to coalesce on a topic and for the partnership team to seed and facilitate interactions.
+
+(cl-academic-industry-community-engagement)=
+### Identifying 'Quick Win' Engagement Opportunities
+A useful activity before jumping in to create new engagement opportunities is to explore what already exists at each organisation. 
+For example there may be a seminar series, journal club or working group at one of the organisations that colleagues from the other are welcome to join.
+
+For the Turing-Roche Partnership, Roche colleagues were encouraged to get involved with the [Turing Institute Interest Groups](https://www.turing.ac.uk/research/interest-groups) which are open for anyone to join and a good way to get a sense of what research is being explored at the Turing. 
+On the Roche side, the team explored areas of the business and prepared case studies of where potential partnership research could be applied- gaining buy-in from colleagues.  
+
+(cl-academic-industry-community-champions)=
+### Identifying Champions/Cheerleaders on Both Sides 
+Whilst there will likely be a core team in a collaboration, it's also useful to identify people outside of this team at both organisations who can champion your collaboration and help raise awareness. 
+This may be communications and marketing teams who can help with promotion or 'influential' leaders in relevant departments who can advocate for people getting involved. 
+There may also be existing networks that you can leverage. 
+
+It may also be fruitful to consider an external advisory board. 
+You can bring on board individuals who can help advise on scientific strategy and also can connect the collaboration with colleagues and opportunities within their own networks.
+
+(cl-academic-industry-community-space)=
+### Creating Shared Spaces to Connect
+To start facilitating interactions and connections between your organisations and allow your community to come together its worthwhile to set up a shared space where people can interact. 
+This could be something like a Slack Workspace or GitHub Repository which are both fairly low barrier to engage with. 
+More info on GitHub this {ref}`the Getting Started with GitHub chapter <cl-github-novice>`. 
+You might need to check any tool you're considering with your IT department and take into consideration any restrictions/access issues each organisation might have.
+
+As your collaboration grows and progresses you may also consider joint events, such as a seminar series, journal club or joint conference/symposium. 
+Collaboration Cafes and/or co-working sessions can also help bring a sense of community to your collaboration. 
```

Comment:
worth adding a ref to the Collab Cafe page?
https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html